source/fuzz/pass_management/repeated_pass_recommender_standard.cpp

```diff
@@ -220,6 +220,11 @@ RepeatedPassRecommenderStandard::GetFuturePassRecommendations(
     //   different way
     return RandomOrderAndNonNull({pass_instances_->GetSplitBlocks()});
   }
+  if (&pass == pass_instances_->GetMergeFunctionReturns()) {
+    // - Functions without early returns are more likely to be able to be
+    //   inlined.
+    return RandomOrderAndNonNull({pass_instances_->GetMergeFunctionReturns()});
```

Comment:
This should recommend inlining functions, not doing more merging.

Reply:
I also think it would be worth recommending "merge function returns" after "donate modules", as donated functions may be good candidates for having their returns merged.

Reply:
Fixed